content/en/data_streams/_index.md

```diff
@@ -59,6 +59,16 @@ High lag on a consuming service, increased resource use on a Kafka broker and in
 
 Click on the **Throughput** tab on any service or queue in Data Streams Monitoring to quickly detect changes in throughput, and which upstream or downstream service these changes originate from. Once the [Service Catalog][2] is configured, you can immediately pivot to the corresponding team's Slack channel or on-call engineer.
 
+### Pivot from pipeline health to slowest individual messages
+
+Delays in your pipeline can also be due to increased processing latency for messages within a consuming service. A faulty deploy or an unexpected change in message format can cause end-to-end latency to spike.
+
+{{< img src="data_streams/data_streams_service_health_tab.png" alt="Datadog Data Streams Monitoring" style="width:100%;" >}}
```

Comment:
This image isn't rendering in the preview. It doesn't look like it exists on master. Could you add it to the PR at `static/images/data_streams`?

Reply:
Added it, sorry for the delay.